FunctionTesting/src/ro/sci/bv/CelsiusConverter/CelsiusConverter.java

```diff
@@ -0,0 +1,52 @@
+package ro.sci.bv.CelsiusConverter;
+import java.text.DecimalFormat;
+import java.util.Scanner;
+
+/**
+ * This program prompts the user to insert a temperature in Fahrenheit degrees
+ * Then it realizes a conversion to Celsius degrees and outputs a message, based on the inputted value
+ *
+ * @author Bogdan Butuza
+ * Modify date 02/03/2019
+ */
+public class CelsiusConverter {
+
+    /**
+     * This method converts a temperature from Fahrenheit degrees to Celsius degrees
+     * @param temperature a double value, representing the temperature to be converted
+     * @return the converted temperature in Celsius degrees, a double value
+     */
+    public double convertCelsius(double temperature) {
+        return ((temperature - 32) * 5/9);
+    }
+
+    /**
+     * This method is used to output a message to the user, regarding the given temperature
+     * @param temperature receives the converted temperature, as a double value
+     * @return a message by every case: normal temperature, illness or an invalid temperature for the human body
+     */
+    public String getMessage(double temperature) {
+        if ((temperature >= 36.20) && (temperature <= 45)) {
+            //Normal temperature values for humans are between 36.20 and 37.20
+            if (temperature >= 37.30) {
+                return "You are ill!";
+            }
+            return "Your temperature is normal.";
+        }else{
+            return "The given temperature isn't for a human body.";
+        }
+    }
```

Comment:
nice looking code. Clear methods, with javadoc, single responsibility for each method, easy to read. Good job